src/app/shared/popup/duplicate-experience/duplicate-experience.component.spec.ts

```diff
@@ -51,68 +51,68 @@ describe('DuplicateExperienceComponent', () => {
   describe('for select()', () => {
     it('select all when all selected', () => {
       component.roleSelected = {
-        admin: true,
+        author: true,
```

Comment:
I had to update this whole file because it was failing the tests. I don't really have context but doing this seem to fix it cc @shawnm0705